src/Representation/Group.php

```diff
@@ -43,6 +44,9 @@ public function __construct(
         protected ?array $realmRoles = null,
         #[Since('23.0.0')]
         protected ?int $subGroupCount = null,
+        #[Since('23.0.0')]
+        protected ?string $parentId = null,
```

Comment:
Please add a getter and setter to the class's annotation (adhere to the order: getters first, setters second, alphabetical order).